packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.3.1
```

Comment:
Did you use the repository tooling with `--minimal` to create these changes? This shouldn't have a version bump since it only affects test code, and there are other packages in the same situation.

The ones that change `example/main.dart` will get version bumps from the tool since that file is shown on pub.dev, as will the rare ones with actual `lib/` changes, but not everything in this PR changes those.

Reply:
fixed this with commit "version bump"(https://github.com/flutter/plugins/commit/e472173c602a81fdd0a3cb9a69c37f814fb53f2f)